blogs/2020/06/09/go-extension.md

```diff
@@ -41,3 +41,5 @@ From all of us in the Visual Studio Code and Go teams, we are excited for the fu
 Happy Coding!
 
 The VS Code Team
+
+PS: You can read the announcement from the Go team [here](https://blog.golang.org/vscode-go)
```

Comment:
How about:
"P.S. You can also read the announcement from the Go team on [The Go Blog](https://blog.golang.org/vscode-go)."

Using [here] isn't good for screen readers, since it just say "here link" and people won't know where they are jumping off to.

Reply:
Thanks for this, I didn't know that and I learnt something new and important :)